api/v1beta1/vmstaticscrape_types.go

```diff
@@ -63,6 +63,9 @@ type TargetEndpoint struct {
 	// Timeout after which the scrape is ended
 	// +optional
 	ScrapeTimeout string `json:"scrapeTimeout,omitempty"`
+	// MaxScrapeSize defines a maximum size of scraped data for a job
+	// +optional
+	MaxScrapeSize string `json:"max_scrape_size,omitempty"`
```

Comment:
Btw, WDYT of including it into the vm specific params `VMScrapeParams`?

Reply:
updated